### PR TITLE
Smartish content-headers

### PR DESCRIPTION
**Pull request by shepwalker**
Browsers are apparently smart enough to know a jpeg is a png. Twillio is not, so we can't be that lazy.

### CODE REVIEW

File: corji/utils.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # This Python file uses the following encoding: utf-8
+import imghdr
```
Comment (jmduke) [inline corji/utils.py]: remove blankline

File: corji/utils.py [hunk 2]
```diff
@@ -35,6 +36,20 @@ def emoji_contains_skin_tone(text):
 
     return text[1] in skin_tone_characters
 
+
+def get_content_type_header(request):
+    detected_content_type = imghdr.what("blerg", h=request.content)
```
Comment (jmduke) [inline corji/utils.py]: "blerg"?

File: corji/utils.py [hunk 3]
```diff
@@ -35,6 +36,20 @@ def emoji_contains_skin_tone(text):
...
+    content_header = detected_to_header_mapping.get(
+        detected_content_type, None)
+    if not content_header:
+        if request.headers['content-type'] in accepted_mime_types:
+            return request.headers['content-type']
+        else:
```
Comment (jmduke) [inline corji/utils.py]: add comment explaining we use jpeg as the fallback

File: corji/utils.py [hunk 4]
```diff
@@ -54,3 +69,11 @@ def emoji_contains_skin_tone(text):
     '>_<': '😣',
     'B-)': '😎'
 }
+
+accepted_mime_types = ['image/png', 'image/jpeg', 'image/jpg', 'image/gif']
+
+detected_to_header_mapping = {
```
Comment (jmduke) [inline corji/utils.py]: what about `.jpg`?

File: corji/utils.py [hunk 5]
```diff
@@ -35,6 +36,20 @@ def emoji_contains_skin_tone(text):
 
     return text[1] in skin_tone_characters
 
+
+def get_content_type_header(request):
```
Comment (jmduke) [inline corji/utils.py]: add tests to `tests/utils.py` for this, handling all the obvious cases

File: corji/utils.py (hunk 4 above)
Comment (shepwalker) [inline corji/utils.py]: https://docs.python.org/2/library/imghdr.html This dict is used to map the
output of the .what() call to a header name.  'jpg' is not a possible
response from imghdr.what().

File: corji/utils.py (hunk 2 above)
Comment (shepwalker) [inline corji/utils.py]: imghdr.what() has one mandatory argument that maps points to the file that
the look-up is being done against (ie, point to a photo on the local box).
This argument gets completely ignored if the 'h' keyword is populated.
Really lovely design.

File: corji/utils.py (hunk 3 above)
Comment (shepwalker) [inline corji/utils.py]: Done. Shouldn't AutoPep8 find this kind of thing?

**Pull request merged**